src/libultra/io/dpsetstat.c

```diff
@@ -1,4 +1,5 @@
-#include "global.h"
+#include "rcp.h"
+#include "rdp.h"
```

Comment:
```suggestion
```
?
and dpgetstat

Reply:
`rdp.h` contains the prototype for `osDpSetStatus`, the header containing the prototype needs to be included so the compiler can warn/error if the prototype does not match the function itself